be/src/olap/rowset/segment_v2/column_writer.h

```diff
@@ -108,8 +108,7 @@ class ColumnWriter {
     struct Page {
         int32_t first_rowid;
         int32_t num_rows;
-        OwnedSlice null_bitmap;
-        OwnedSlice data;
+        std::vector<OwnedSlice> data;
```

Comment:
It will be better to introduce an OwnedSlice named compressed_data. If you reuse the data struct, it will make this struct difficult to understand.

Reply:
Because the compression may not compress data, and the data vector will contain header/null bitmap/uncompressed data OwnedSlice, so use vector will be more easy.